### PR TITLE
Update release checklist, for recent changes following Jan CPU

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes : #211 

This PR is required to update the release checklist, to cover new steps, and amendments to the release process.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (zdtsw) [inline .github/ISSUE_TEMPLATE/release-checklist.md:99]: maybe mention it is standard nightly and the evaluation nightly both

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (zdtsw) [inline .github/ISSUE_TEMPLATE/release-checklist.md:115]: some more description: where/how to check, if/which account to login nagios UI and how to report if issue

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (zdtsw) [inline .github/ISSUE_TEMPLATE/release-checklist.md:125]:
```suggestion
- [ ] **Add website banner** (this is done by making a PR then it gets automate published to website) - Announce that we target releases to be available within 48-72 hours of the GA tags being available
```

**Comment by zdtsw** [comment]
some extra thoughts:
1. we should specify there is a different properties file for jdk8 arm32 linux to update
2. last time we talked about to make these release branches to be protected(need PR approval before merge). not sure if it is done, wrothy have a test. 
3. have we done the auto trigger build for jdk8 arm32? should be an issue on this

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jiekang) [reply .github/ISSUE_TEMPLATE/release-checklist.md:99]: As new reader, question comes to mind: How do I disable the pipelines? 

Also might be worth explicitly listing pipeline names all together (for nightly & weekly). E.g replace disable nightly/weekly with just:

Disable the following pipelines:
...

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jiekang) [reply .github/ISSUE_TEMPLATE/release-checklist.md:57]: It's mentioned in L108 "Generate release pipeline jobs..."; should it be moved earlier? Or is it a different instruction?

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jiekang) [inline .github/ISSUE_TEMPLATE/release-checklist.md:133]: Think this needs some structural adjustment. Naively it reads everything following this statement is automated all the way to declaration of the release as complete.

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (steelhead31) [reply .github/ISSUE_TEMPLATE/release-checklist.md:28]: Issue #212 has been raised

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (steelhead31) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: Adjusted appropriately, to reduce confusion.

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (steelhead31) [reply .github/ISSUE_TEMPLATE/release-checklist.md:117]: Done, theres also some presentation and documentation on this, that is currently being written, once its done, I'll update this checklist to point at the "How-To" guide

**Comment by zdtsw** [comment]
occurred to me two more items:
1. create retro issues in adoptium and compliance repo
2. update bookmarks in slack channels for correct release

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (andrew-m-leonard) [inline .github/ISSUE_TEMPLATE/release-checklist.md:101]: Should add a note to do this by setting defult for ENABLE_PIPELINE_SCHEDULE to false in the https://ci.adoptium.net/job/build-scripts/job/utils/job/build-pipeline-generator/, and then re-running it.

Comment (andrew-m-leonard) [reply .github/ISSUE_TEMPLATE/release-checklist.md:101]: otherwise if you just disable the project, upon a PR merge regen will re-enable it...

Comment (zdtsw) [reply .github/ISSUE_TEMPLATE/release-checklist.md:101]: Yes, the proper way is to do it from config and re-run seed job.

Last time, I did the shortcut to only disabled it from Jenkins UI, based on no new commit could be merged to "master" of ci-jenkins-pipeline during the release period.  
But since we are using release branches now, it is safer to do it from config.

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (andrew-m-leonard) [inline .github/ISSUE_TEMPLATE/release-checklist.md:102]: ditto ^

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (andrew-m-leonard) [inline .github/ISSUE_TEMPLATE/release-checklist.md:116]: I think the cleaning job needs running at the 2week point, as we are likely to hit "locked" files, as Shelley mentioned...

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (steelhead31) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: Done!

**Review by andrew-m-leonard — APPROVED** [review]

**Pull request merged**
